The `complete` app does not build

Not sure If I'm missing something here, but I cloned the repo, moved to the complete dire, and ran `./gradlew build` and git hit by basic dependency resolution issues :

```
 ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-6.5-bin.zip
.................................................................................................

Welcome to Gradle 6.5!

Here are the highlights of this release:
 - Experimental file-system watching
 - Improved version ordering
 - New samples

For more details see https://docs.gradle.org/6.5/release-notes.html

Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details

> Task :complete:compileJava FAILED
/Users/jgjoseph/dev/micronaut-creating-first-graal-app/complete/src/main/java/example/micronaut/Application.java:3: error: package io.micronaut.runtime does not exist
import io.micronaut.runtime.Micronaut;
                           ^
/Users/jgjoseph/dev/micronaut-creating-first-graal-app/complete/src/main/java/example/micronaut/ConferenceService.java:3: error: package javax.inject does not exist
import javax.inject.Singleton;
                   ^
/Users/jgjoseph/dev/micronaut-creating-first-graal-app/complete/src/main/java/example/micronaut/ConferenceService.java:8: error: cannot find symbol
@Singleton // <1>
 ^
  symbol: class Singleton
/Users/jgjoseph/dev/micronaut-creating-first-graal-app/complete/src/main/java/example/micronaut/Conference.java:3: error: package io.micronaut.core.annotation does not exist
import io.micronaut.core.annotation.Introspected;
```

Adding the native Image task to the [basic application](https://github.com/micronaut-guides/creating-your-first-micronaut-app) creates a native image for that though.

--- a/complete/gradle.properties
+++ b/complete/gradle.properties
@@ -1,1 +1,1 @@
-micronautVersion=2.3.0
+micronautVersion=2.3.4

--- a/complete/gradle.properties
+++ b/complete/gradle.properties
@@ -1,1 +1,1 @@
-micronautVersion=2.3.4
+micronautVersion=2.4.0

